lib/tasks/travis.rake

```diff
@@ -0,0 +1,11 @@
+require 'system_commands'
+
+namespace :travis do
+  desc "Update vendor/bundle files to gem cache on tmc-server"
+  task :gemcache do
+    puts "Pushing new gems from local gem cache (vendor/bundle) to travis cache"
+    SystemCommands.sh!("find vendor/bundle -iname '*.gem'  -print0  | rsync -0 --checksum -avz  --files-from=- . web01:/srv/www/testmycode.net/travis/rubygems-cache/", {escape: false})
```

Comment:
Good points both - I feel that having the server address set here is too specific to our setup...
Thou I don't feel like it should go to site.yml either. Maybe setting the host from env, but it diminish ease of this.

Or maybe just have it specifying the server (web01.testmycode.net) and our path for now.